cg/services/quality_controller/quality_checks/utils.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+from cg.constants.constants import PrepCategory
+from cg.constants.priority import Priority
+from cg.store.models import Case, Sample
+
+LOG = logging.getLogger(__name__)
+
+
+def case_pass_sequencing_qc(case: Case) -> bool:
+    """
+    Get the standard sequencing QC of a case. If the case is express priority, the express QC is
+    used. If the case is a ready made library, the ready made library QC is used.
+    """
+    if is_case_ready_made_library(case):
+        return ready_made_library_case_pass_sequencing_qc(case)
+    if is_case_express_priority(case):
+        return express_case_pass_sequencing_qc(case)
+    return all(sample_has_enough_reads(sample) for sample in case.samples)
+
+
+def express_case_pass_sequencing_qc(case: Case) -> bool:
+    """
+    Checks if all samples in an express case have enough reads.
+    """
+
+    return all(express_sample_has_enough_reads(sample) for sample in case.samples)
+
+
+def express_sample_pass_sequencing_qc(sample: Sample) -> bool:
+    return express_sample_has_enough_reads(sample)
+
+
+def sample_pass_sequencing_qc(sample: Sample) -> bool:
+    """
+    Get the standard sequencing QC of a sample.
+    """
+    if is_sample_express_priority(sample):
+        return express_sample_pass_sequencing_qc(sample)
+    if is_sample_ready_made_library(sample):
+        return ready_made_library_sample_has_enough_reads(sample)
```

Comment:
Same here with the order of the checks, here in addition we are evaluating `sample_pass_sequencing_qc` checks first 🤔